components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/FtpConfiguration.java

```diff
@@ -83,4 +85,16 @@ public void setChmod(String chmod) {
     public String getChmod() {
         return chmod;
     }
+
+    public boolean isDirectoryParseResultAbsolute() {
+        return isDirectoryParseResultAbsolute;
+    }
+
+    /**
+     * Allows you to set how the consumer will handle subfolders and files
+     * in the path if the directory parser results in with absolute paths
```

Comment:
Maybe add a comment: Some FTP servers may return file names with absolute paths, and if so then the FTP component needs to handle this by converting the returned path into a relative path.